mmpose/models/heads/base_head.py

```diff
@@ -63,9 +63,14 @@ def _get_in_channels(self) -> Union[int, List[int]]:
 
         return in_channels
 
-    def _transform_inputs(self, feats: Tuple[Tensor]
-                          ) -> Union[Tensor, Tuple[Tensor]]:
+    def _transform_inputs(
+        self,
+        feats: Union[Tensor, Sequence[Tensor]],
+    ) -> Union[Tensor, Tuple[Tensor]]:
         """Transform multi scale features into the network input."""
+        if not isinstance(feats, Sequence):
+            return feats
```

Comment:
I think it's better to warn the users that inputting a bare tensor doesn't meet our convention and the `input_transform` argument will be ignored.

Reply:
OK, the warning has been added